src/helpers/getByAPI.js

```diff
@@ -17,12 +17,23 @@ const getNodeByText = (node, text) => {
   try {
     // eslint-disable-next-line
     const { Text, TextInput } = require('react-native');
-    return (
-      (filterNodeByType(node, Text) || filterNodeByType(node, TextInput)) &&
-      (typeof text === 'string'
-        ? text === node.props.children
-        : text.test(node.props.children))
-    );
+    const isTextNode =
+      filterNodeByType(node, Text) || filterNodeByType(node, TextInput);
+    if (isTextNode) {
```

Comment:
I don't love how this came out, so I'd love some feedback here!

Reply:
I think that's fine. Can we change the naming to `isTextComponent`?

Reply:
Done in 7a1a3f4